recipes/quilt/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "quilt3" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/quiltdata/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: eec5bee2d0037df8b726b88fba373596e3fc7a15f473013faeef20a5cb0415b1
```

Comment:
Did I do this wrong?
```
(glam) apk-mbp-3:t4 karve$ shasum -a 256 ~/Downloads/quilt-3.0.5.tar.gz 
ded69d039949de8293b0c98e95cd0a958eb7e68c93106b1f35254dd4691685c4
```

Reply:
no, you're right, not sure where i got that other hash